ombt2

```diff
@@ -396,8 +438,13 @@ class _TestClient(_Client):
 class _TestServer(_Client):
     """Base class for Listener and RPC servers
     """
-    def __init__(self, cfg, ctl_url, topic, kind, timeout):
-        super(_TestServer, self).__init__(cfg, ctl_url, topic, kind, timeout)
+    def __init__(self, cfg, ctl_url, topic, kind, name, timeout, output=None):
+        super(_TestServer, self).__init__(cfg, ctl_url, topic, kind, timeout,
+                                          name, output)
+        self._output.write('{"test_name": "UNKNOWN",\n')
+        self._output.write(' "start": %f,\n' % now())
+        self._output.write(' "data": [\n')
```

Comment:
Maybe each line can be a valid JSON like suggested by http://jsonlines.org/

Reply:
That is a good idea.  If this were structured json we'd likely hit some issues trying to parse the entire dataset as a single json structure if the data set is huge.